config/initializers/highlight_override.rb

```diff
@@ -60,9 +60,9 @@ def create_model!(user_uuid:)
 end
 
 Api::V0::Bindings::Highlights.class_exec do
-  def self.create_from_models(highlights, pagination)
+  def self.create_from_models(highlights, pagination, total_count)
     highlights = highlights.to_a unless highlights.is_a? Array
-    meta = pagination.merge(total_count: highlights.count)
+    meta = pagination.merge(total_count: total_count)
     attribs = { meta: meta, data: highlights }
     highlights_response = new(attribs)
```

Comment:
The `highlights` passed in here is an array of ActiveRecords.  Needs to be an array of `Api::V0::Binding::Highlight`.

Reply:
fixed.